lib/mssql.js

```diff
@@ -318,12 +318,11 @@ MsSQL.prototype.destroyAll = function (model, where, cb) {
     cb = where;
     where = undefined;
   }
-  this.query('DELETE FROM '
-    + this.tableEscaped(model) + ' ' + this.buildWhere(model, where || {}),
-    function (err, data) {
-    cb && cb(err, data);
-  }.bind(this));
-
+  var sql = 'DELETE FROM ' + this.tableEscaped(model) + ' ' +
+      this.buildWhere(model, where || {}) + ';SELECT @@ROWCOUNT as count';
+  this.query(sql, function(err, info) {
+    cb && cb(err, info[0]);
```

Comment:
Is this assuming that `info[0]` is an object with a `count` property? I find it a bit cryptic, can you perhaps add a comment to explain?